src/shared/definitions.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Clone, PartialEq, Eq, Hash, Debug)]
+pub struct TableId(pub String);
+
+#[derive(Clone, PartialEq, Eq, Hash, Debug)]
+pub struct PageId(pub usize);
```

Comment:
Page ID will always be 4 bytes ([ref](https://docs.google.com/document/d/1cxzR6UgDiBOHNMSnQcVqE9SaSkJJe5dxPUKDHtnDpkY/edit))

let's change this to u32